generators/browser-compatible-base/src/ast/AbstractWriter.ts

```diff
@@ -150,6 +150,24 @@ export class AbstractWriter {
         this.indentLevel--;
     }
 
+    public delimit<T extends AbstractAstNode>(
+        nodes: T[],
+        delimiter: string,
+        writeFunction: (node: T) => void,
+        prefix: string = "",
```

Comment:
Let's make sure to fix these `eslint` errors.

Reply:
Yep, chugging away at them now...